src/misc/lv_math.h

```diff
@@ -27,6 +27,9 @@ extern "C" {
 #define LV_BEZIER_VAL_MAX (1L << LV_BEZIER_VAL_SHIFT) /**< Max time in Bezier functions (not [0..1] to use integers)*/
 #define LV_BEZIER_VAL_FLOAT(f) ((int32_t)((f) * LV_BEZIER_VAL_MAX)) /**< Convert const float number cubic-bezier values to fix-point value*/
 
+/*Align up value x to align, align must be a power of two*/
+#define LV_ALIGN_UP(x, align) (((x) + ((align) - 1)) & ~((align) - 1))
```

Comment:
In general, isn't it `LV_ROUND_UP`?

Reply:
It’s called align up in NuttX and tlsf https://github.com/lvgl/lvgl/blob/08803f9a115f845417513a2b9bb3ae251212f968/src/stdlib/builtin/lv_tlsf.c#L484C21-L484C21

Reply:
Ah, `round_up` should handle negative numbers too, while `align_up` has a `size_t` parameter. 

Ok, let's use it.